app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/DutiesSelectedAnswer.scala

```diff
@@ -16,60 +16,35 @@
 
 package uk.gov.hmrc.cdsreimbursementclaimfrontend.models
 
-import cats.kernel.Eq
-import julienrf.json.derived
-import play.api.libs.json.OFormat
-import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.form.{DutiesSelected, Duty}
-
-sealed trait DutiesSelectedAnswer extends Product with Serializable
+import cats.data.NonEmptyList
+import play.api.libs.json.Format
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.form.Duty
 
 object DutiesSelectedAnswer {
 
-  final case class IncompleteDutiesSelectedAnswer(
-    maybeDutiesSelected: Option[DutiesSelected]
-  ) extends DutiesSelectedAnswer
-
-  object IncompleteDutiesSelectedAnswer {
-    val empty: IncompleteDutiesSelectedAnswer           = IncompleteDutiesSelectedAnswer(None)
-    implicit val eq: Eq[IncompleteDutiesSelectedAnswer] = Eq.fromUniversalEquals[IncompleteDutiesSelectedAnswer]
+  type DutiesSelectedAnswer = NonEmptyList[Duty]
```

Comment:
@gabor-dorcsinecz Is it possible to have an apply method like this?
```
def apply(duty: Duty): DutiesSelectedAnswer = NonEmptyList.of(duty)
```
Then in the code we can change 
```
val dutiesSelectedAnswer = NonEmptyList.of(Duty(TaxCode.A00))
```
to
```
val dutiesSelectedAnswer = DutiesSelectedAnswer(Duty(TaxCode.A00))
```
Which perhaps is more convenient? But not mandatory

Also would it be better rather than create 
`object DutiesSelectedAnswer  { type DutiesSelectedAnswer = NonEmptyList[Duty] }` 
to put this `type DutiesSelectedAnswer = NonEmptyList[Duty]`
inside existing `models` package file like?
```
package uk.gov.hmrc.cdsreimbursementclaimfrontend

package object models {
 type DutiesSelectedAnswer = NonEmptyList[Duty]

 object DutiesSelectedAnswer(duty: Duty) {
    def apply(duty: Duty): DutiesSelectedAnswer = NonEmptyList.of(duty)
 }

 // so that next similar alias can be listed below
}
```
Also this approach does not require you to provide an import explicitly if you wish. to use an alias within code.
Presumably having an import
`import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.form.Duty`
will auto import aliases from models already.
But the current one approach does require you to import `uk.gov.hmrc.cdsreimbursementclaimfrontend.models.DutiesSelectedAnswer._` to use an alias. 

Ones again it is very subjective, @mikail-khan @caoilte what is your thoughts?